client.go

```diff
@@ -78,7 +78,7 @@ func NewClient(config *ClientConfig) (*Client, error) {
 
 	if config.Net == nil {
 		config.Net = vnet.NewNet(nil) // defaults to native operation
-	} else {
+	} else if config.Net.IsVirtual() {
```

Comment:
should we do this for ice as well? (https://github.com/pion/ice/blob/master/agent.go#L345)

Reply:
Yes, I have [this commit](https://github.com/pion/ice/pull/103/commits/844561e47b58c47b1c95c9c5dd90219fb31c56ad)